global/classic-zaza/requirements.txt

```diff
@@ -13,5 +13,5 @@ netifaces>=0.10.4
 netaddr>=0.7.12,!=0.7.16
 Jinja2>=2.6  # BSD License (3 clause)
 six>=1.9.0
-dnspython>=1.12.0
+dnspython>=1.12.0,<2.0.0  # dnspython dropped py3.5 support
```

Comment:
Can we do:

```
dnspython>=1.12.0,<2.0.0 ; python_version < '3.6'
dnspython>=1.12.0 ; python_version >= '3.6' 
```

See https://www.python.org/dev/peps/pep-0508/#environment-markers for more details.